Sources/SwiftWin32/CG/AffineTransform.swift

```diff
@@ -59,6 +59,18 @@ public struct AffineTransform {
                            tx: self.tx * transform.a + self.ty * transform.c + transform.tx,
                            ty: self.tx * transform.b + self.ty * transform.d + transform.ty)
   }
+
+  public func inverted() -> AffineTransform {
+    let determinant = a * d - b * c
+    let (newA, newB, newC, newD) = (d/determinant, -b/determinant,
+                                   -c/determinant,  a/determinant)
+    let (newTX, newTY) = (-newA * tx - newC * ty, -newB * tx - newD * ty)
+    let invertedTransform = 
+      AffineTransform( a: newA,  b: newB,
+                       c: newC,  d: newD,
+                      tx: newTX, ty: newTY)
+    return invertedTransform
```

Comment:
```suggestion
    let determinant = self.a * self.d - self.b * self.c
  
    let a = self.d / determinant
    let b = -self.b / determinant
    let c = -self.c / determinant
    let d = self.a / determinant
  
    return AffineTransform(a: a, b: b, c: c, d: d,
                           tx: -a * self.tx * c * self.ty,
                           ty: -b * self.tx - d * self.ty)
    return invertedTransform
```

Seems a bit more concise and avoids an unnecessary set of tuples to be created.  The spacing also makes it easier to follow.